internal/integration/base/api.go

```diff
@@ -167,11 +169,15 @@ func (apiSuite *APISuite) Capabilities() Capabilities {
 		}
 	}
 
-	if _, err = apiSuite.Client.LS(context.Background(), &machineapi.ListRequest{
-		Root:  sdboot.SystemdBootStubInfoPath,
-		Types: []machineapi.ListRequest_Type{machineapi.ListRequest_REGULAR},
-	}); err == nil {
-		caps.TrustedBoot = true
+	rList, err := apiSuite.Client.COSI.List(context.Background(), resource.NewMetadata(v1alpha1.NamespaceName, runtimeresources.SecurityStateType, "", resource.VersionUndefined))
```

Comment:
I wonder if we should also have the input to the test (test flag) which says it's expected to be secureboot

so that we can move the check from the shell script to the integration test - if expected to secureboot (or vice versa), the resource should appear

Reply:
yes, i see that makes things easier